app/src/js/AstroVectorGrid.js

```diff
@@ -146,9 +146,47 @@ export default L.AstroVectorGrid = L.VectorGrid.Protobuf.extend({
       }
     }
     this.changeSelectedStyle(highlightedFeatures);
+    this._selectedFeatures=highlightedFeatures;
```

Comment:
changeSelectedStyle() should add the highlighted features to this._selectedFeatures. Adding them here is adding them twice and idk if that causes any problems.